AirCasting/SessionViews/ChartDatabaseObserver.swift

```diff
@@ -0,0 +1,59 @@
+// Created by Lunar on 06/02/2022.
+//
+
+import Foundation
+import CoreData
+import Resolver
+
+enum ChartDatabaseObserverFilter {
+    case none
+    case hour
+    case minute
+}
+
+final class ChartDatabaseObserver {
+    @Injected private var persistence: PersistenceController
+    private let onMeasurementsChange: (() -> Void)
+    private let session: SessionUUID
+    private let sensor: String
+    private let filteringComponent: Calendar.Component?
+    
+    private var lastMeasurementChangeAt: Date?
+    
+    init(session: SessionUUID, sensor: String, timedFilter: ChartDatabaseObserverFilter, onMeasurementsChange: @escaping (() -> Void)) {
```

Comment:
What does `timedFilter` mean? Should it be `timeFilter`?

Reply:
changed to `filtered`